src/Tribe/Aggregator/Page.php

```diff
@@ -343,7 +360,7 @@ public function template( $name, $data = array(), $echo = true ) {
 		 * @param string $name     Template name
 		 * @param array  $data     The Data that will be used on this template
 		 */
-		do_action( 'tribe_aggregator_template_before_include', $file, $name, $data );
+		do_action( 'tribe_aggregator_template_before_include', $file, $name, $data ); // phpcs:ignore
```

Comment:
Move the `phpcs:ignore` comment up, not inline.
We try to avoid inline comments, if possible.

```suggestion
		 // phpcs:ignore
		do_action( 'tribe_aggregator_template_before_include', $file, $name, $data );
```

What is the reason for this phpcs ignore?